src/main/java/jenkins/branch/EmptyOrganizationFolderView.java

```diff
@@ -0,0 +1,67 @@
+package jenkins.branch;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Collections;
+
+import javax.servlet.ServletException;
+
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+
+import hudson.model.Item;
+import hudson.model.TopLevelItem;
+import hudson.model.View;
+import hudson.model.Descriptor.FormException;
+
+/**
+ * Special view used when {@link OrganizationFolder} has no repositories.
+ *
+ * This view shows the UI to guide users to set up the repositories.
+ */
+public class EmptyOrganizationFolderView extends View {
+    /**
+     * Constructor
+     */
+    public EmptyOrganizationFolderView(OrganizationFolder owner) {
+        super("..", owner);
```

Comment:
(It works fine, so it must be 'legal'). Could you explain how to fix the failing test when this is "Welcome", please?